doc/acknowledgements.md

```diff
@@ -35,8 +35,6 @@ projects. This is not an exhaustive list -- if your project is listed in our
 * [cmocean](https://github.com/matplotlib/cmocean)
 * [Python](https://www.python.org)
 * [Luigi](https://github.com/spotify/luigi)
-* [Rasterio](https://github.com/mapbox/rasterio)
```

Comment:
We no longer depend on these packages. We may again one day, but for now we do all the computation at the CLI with `gdal`